app/models/video.rb

```diff
@@ -17,13 +17,13 @@ def self.search_by_title(search_title)
   end
 end
 
-def average_rating 
-  if self.reviews.average(:rating)
-    average = self.reviews.average(:rating)
-  else
-    average = 0
-  end
-end
+#def average_rating 
+#  if self.reviews.average(:rating)
+#    average = self.reviews.average(:rating)
+#  else
+#    average = 0
+#  end
+#end
```

Comment:
Commented code is almost always unnecessary code. This is in the decorator now, so you can delete it from here.